components/org.wso2.carbon.identity.application.authenticator.google/pom.xml

```diff
@@ -92,7 +96,12 @@
 
                             org.wso2.carbon.ui; version="${carbon.kernel.package.import.version.range}",
                             net.minidev.json; version="${net.minidev.json.imp.pkg.version.range}",
+                            org.wso2.carbon.idp.mgt; version="${carbon.identity.framework.package.import.version.range}",
                         </Import-Package>
+                        <Embed-Dependency>
+                            google-api-client,
+                            org.wso2.carbon.identity.application.authenticator.oidc
```

Comment:
we don't need to embed the oidc authenticator. It will be available in the runtime

Reply:
Fixed 389118ef7cce4b6917eb5b97117e2badacdcf060